### PR TITLE
fix docker version

### PR DESCRIPTION
**Pull request by p365labs**
This PR try to solve the problem I show on issue #1406

### CODE REVIEW

**Comment by p365labs** [comment]
@ruflin If u merge this I have also a branch to remove the CrossIndex class before create the release :)

File: .travis.yml [hunk 1]
```diff
@@ -25,7 +25,7 @@ before_install:
 #  - sudo apt-cache madison docker-engine
 
   # update docker engine to the desired version
-  - sudo apt-get -o Dpkg::Options::="--force-confnew" install -y docker-ce=17.07.0~ce-0~ubuntu
+  - sudo apt-get -o Dpkg::Options::="--force-confnew" install -y docker-ce=17.09.0~ce-0~ubuntu
```
Comment (ruflin) [inline .travis.yml]: I'm confused. Why is this even a diff? This is what I did in my commit: https://github.com/ruflin/Elastica/blob/master/.travis.yml#L28

You had a nicer version in your previous PR where it was a variable.

**Comment by ruflin** [comment]
@p365labs Great. Lets wait for the CrossIndex removal PR then.

File: .travis.yml (hunk 1 above)
Comment (p365labs) [reply .travis.yml]: yes with the ENV variables

**Comment by ruflin** [comment]
@p365labs That looks much better, but now you need to rebase :-)

**Comment by p365labs** [comment]
@ruflin  have a look it now :)

**Comment by p365labs** [comment]
you still have CrossIndex PR #1411

**Comment by ruflin** [comment]
Thanks, left a minor comment on #1411

**Pull request merged**